src/main/java/org/jfrog/gradle/plugin/artifactory/utils/PluginUtils.java

```diff
@@ -67,6 +68,9 @@ public static void addArtifactoryResolutionRepositoryAction(MavenArtifactReposit
                 credentials.setUsername(username);
                 credentials.setPassword(password);
             });
+
+            // Make the authentication preemptive
```

Comment:
Let's ade to the method description an explanation for why it is important to make the authentication preemptive. 
This can help avoid a situation where this line will be removed in the future without understanding the impact.